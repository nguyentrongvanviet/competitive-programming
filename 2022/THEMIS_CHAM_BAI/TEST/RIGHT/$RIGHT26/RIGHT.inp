71 69
451
316
330
460
539
239
31
369
377
435
477
313
472
13
428
381
443
23
22
334
340
445
311
37
302
297
552
493
442
394
420
425
28
18
545
376
226
480
40
457
1
368
229
402
466
7
280
550
43
387
219
283
393
274
411
347
416
500
405
352
342
324
487
292
332
396
234
356
320
309
288
