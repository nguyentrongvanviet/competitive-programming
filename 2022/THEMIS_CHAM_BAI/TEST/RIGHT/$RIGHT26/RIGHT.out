520
385
468
667
815
308
100
576
653
780
891
451
886
82
773
726
857
161
160
541
616
928
518
244
578
573
1104
1045
856
808
903
977
235
156
1166
721
502
1101
316
1078
70
713
574
885
1156
145
694
1309
388
870
633
766
945
757
1032
899
1106
1259
1026
973
894
876
1246
844
953
1086
717
1046
941
930
840
